### PR TITLE
RunTemplateTests

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Clarify intentions of `RunTemplateTests` setting.

### CODE REVIEW

File: src/ProjectTemplates/test/ProjectTemplates.Tests.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <TestGroupName>ProjectTemplates.E2ETests</TestGroupName>
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
 
-    <RunTemplateTests>true</RunTemplateTests>
+    <RunTemplateTests Condition="'$(RunTemplateTests)' == ''" >true</RunTemplateTests>
```
Comment (javiercn) [inline src/ProjectTemplates/test/ProjectTemplates.Tests.csproj]: `?` These tests already ran in all the platforms. Are you trying to run the Selenium bits? (which only run on windows)

Comment (javiercn) [reply src/ProjectTemplates/test/ProjectTemplates.Tests.csproj]: If that's the case `SeleniumE2ETestsSupported` that's what you are looking to set

Comment (ryanbrandenburg) [reply src/ProjectTemplates/test/ProjectTemplates.Tests.csproj]: No, I'm just trying to clarify that `RunTemplateTests` can be (and is) set elsewhere, and that should be the value if it is.

**Comment by ryanbrandenburg** [comment]
⌚️

**Comment by javiercn** [comment]
This is a draft PR, can you mark it as non-draft? Is this the full scope of the change?

**Comment by ryanbrandenburg** [comment]
Yes and yes.

**Review by javiercn — APPROVED** [review]

**Pull request merged**